R/route.R

```diff
@@ -340,12 +340,12 @@ route_and_rear_deltas <- function(year, month, migrants, north_delta_fish, south
   if (month != 8) {
     north_delta_fish <- rear(juveniles = north_delta_fish$inchannel,
                              survival_rate = rearing_survival_delta[1, ],
-                             growth = growth_rates,
+                             growth = growth_rates[,,1],
```

Comment:
do we want `growth_rates_delta` here instead?

Reply:
nevermind I am now seeing you would pass growth_rates_delta in as the argument but it is called growth_rates within the function.